### ISSUE TITLE
Thoughts on Car-Net Compatibility? 

### ISSUE THREAD

**Issue by chandrasg** [tillsteinbach/VWsFriend#149]
Thanks for releasing this amazing package! Are you considering expanding the compatibility to Car-Net (which is the only option for US cars)?

**Comment by tillsteinbach** [comment]
To be honest, I have no overview at all about what is offered in the US. Is there a us version similar to WeConnect ID?

**Comment by chandrasg** [comment]
Yes, US VW comes with CarNet subscription: carnet.vw.com/ <http://carnet.vw.com/>

There was a project to use their API: https://github.com/thomasesmith/vw-car-net-api <https://github.com/thomasesmith/vw-car-net-api>  
but looks like its no longer working after a 2020 update. 

There is another project to call Car Net through WeConnect: https://github.com/nekman/node-vw-carnet <https://github.com/nekman/node-vw-carnet>

**Comment by tillsteinbach** [comment]
I'm using the new API introduced with the ID family. So I would first at all need to have a chat with an ID.4 owner to understand the current state there.

**Comment by dsmk** [comment]
> I'm using the new API introduced with the ID family. So I would first at all need to have a chat with an ID.4 owner to understand the current state there.

I just got my ID.4 AWD in the U.S. and have CarNet set up.  I would be happy to chat and/or gather some information about CarNet connectivity.

**Comment by tillsteinbach** [comment]
The way it is implemented for Volkswagen US looks pretty different than anything I have seen for european cars. I will find out if I can somehow install the Carnet app in an emulator to see what it is actually doing. That would be the first step for me.

**Comment by kyserv** [comment]
Just offering my assistance / car-net credentials / testing abilities whatever you need

**Comment by tillsteinbach** [comment]
Thank you. Unfortunately the situation is not so easy. I cannot download the App from the AppStore as it is only available in the US. My current plan is to get my hands on an old phone and make an US account for it.
